website/docs/r/bfd_multihop_interface_policy.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_bfd_multihop_interface_policy"
+sidebar_current: "docs-aci-resource-bfd_multihop_interface_policy"
+description: |-
+  Manages ACI BFD Multihop Interface Policy
+---
+
+# aci_bfd_multihop_interface_policy #
+
+Manages ACI BFD Multihop Interface Policy
+
+## API Information ##
+
+* `Class` - bfdMhIfPol
+* `Distinguished Name` - uni/tn-{tn_name}/bfdMhIfPol-{policy_name}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Policies -> Protocol -> BFD Multihop -> Interface Policies
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_bfd_multihop_interface_policy" "example" {
+  tenant_dn             = aci_tenant.example.id
+  name                  = "example"
+  admin_state           = "enabled"
+  annotation            = "orchestrator: terraform"
+  detection_multiplier  = "3"
+  min_transmit_interval = "250"
+  min_receive_interval  = "250"
+}
+```
+
+## Argument Reference ##
+
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object.
+* `name` - (Required) Name of the BFD Multihop Interface Policy object.
+* `annotation` - (Optional) Annotation of the BFD Multihop Interface Policy object.
+* `admin_state` - (Optional) Enable Disable sessions.The administrative state of the object or policy. Allowed values are "disabled", "enabled", and default value is "enabled". Type: String.
+* `description` - (Optional) Description for the BFD Multihop Interface Policy object.
+* `detection_multiplier` - (Optional) Detection Multiplier.Detection multiplier. Allowed range is 1-50 and default value is "3".
+* `min_receive_interval` - (Optional) Required Minimum RX Interval.Required minimum rx interval. Allowed range is 250-999 and default value is "250".
```

Comment:
```suggestion
* `min_receive_interval` - (Optional) Required minimum rx interval. Allowed range is 250-999 and default value is "250".
```

Reply:
Also not sure if we should use `RX` or `rx` or `Rx`

Reply:
Tx/Rx